PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -2920,22 +2949,32 @@ function Get-SddcDiagnosticInfo
         ####
         # Now receive the jobs requiring remote copyout
         ####
+        if($Phase1Timeout -gt [TimeSpan]::Zero)
+        {
+            $Phase1EndTime = (Get-Date) + $Phase1Timeout
+        }
+        else
+        {
+            $Phase1EndTime = $null
+        }

```

Comment:
Should this go up in the vicinity of the "Begin parallelized captures" comment, which is where the gathers really start? There's just enough non-job'd gathers (and/or eventual risk of more, unless you chose to push down on it - perhaps they should be) that their delay may matter at some point.

Reply:
I feel that this is more readable, and the difference between start/end isn't too much, except in cases of extremely short timeout durations, where there is actually a noticeable difference between the first job fired and last job fired, in which you would want this current behavior anyways.